Gallery.Api/Infrastructure/EventHandlers/ExhibitHandler.cs

```diff
@@ -44,14 +44,30 @@ public class ExhibitHandler
             _citeHub = citeHub;
         }
 
-        protected async Task<List<string>> GetMainGroups(ExhibitEntity exhibitEntity, CancellationToken cancellationToken)
+        protected async Task<List<Guid>> GetMainGroups(ExhibitEntity exhibitEntity, CancellationToken cancellationToken)
         {
-            var groupIds = new List<string>();
-            groupIds.Add(exhibitEntity.Id.ToString());
+            var groupIds = new List<Guid>();
+            groupIds.Add(exhibitEntity.Id);
             // add System Admins
-            var systemAdminPermissionId = (await _db.Permissions.Where(p => p.Key == UserClaimTypes.SystemAdmin.ToString()).FirstOrDefaultAsync()).Id.ToString();
+            var systemAdminPermissionId = (await _db.Permissions.Where(p => p.Key == UserClaimTypes.SystemAdmin.ToString()).FirstOrDefaultAsync()).Id;
             groupIds.Add(systemAdminPermissionId);
 
+            // add this exhibit's users
```

Comment:
We don't want to add individual users here.  Users that are viewing the exhibit will get the change when it is sent to the exhibit ID.  See MainHub.cs lines 128 and 129.  No API change is necessary to fix this bug.  Only the UI needs to be changed.